src/Helpers/Build.php

```diff
@@ -285,7 +285,7 @@ protected function _getChannel( $token, $type, $version ) {
          * @param string $fileName The name of the file
          * @return string The absolute URL for the file to be downloaded
          */
-        protected function _getUrl( $fileName = '', $basePath ) {
+        protected function _getUrl( $fileName, $basePath ) {
```

Comment:
Will `$fileName` be empty by default?

Reply:
All references to _getUrl already specified $fileName, so assigning it a default value was unnecessary at this time.  I made the change because it is a syntax error in PHP8 to follow a parameter having a default value with one that lacks a default value, and this seemed the least invasive fix.

Reply:
Understood, did you manage to test this code already?

Reply:
Yes, I tested it with some personal android builds before submitting the PR.